eng/pipelines/templates/jobs/ci.yml

```diff
@@ -159,6 +159,7 @@ jobs:
         - template: /eng/common/pipelines/templates/steps/daily-dev-build-variable.yml
           parameters:
             ServiceDirectory: ${{parameters.ServiceDirectory}}
+            Artifacts: ${{parameters.ReleaseArtifacts}}
```

Comment:
Why is this here? It doesn't even seem to have an Artifacts list.

Reply:
I see you are adding it in https://github.com/Azure/azure-sdk-tools/pull/13202/files